flux_led/__main__.py

```diff
@@ -940,7 +947,29 @@ def setRgbw(self, r=None, g=None, b=None, w=None, persist=True,
                 msg.append(int(w))
             else:
                 msg.append(int(0))
-            msg.append(special)
+
+            if self.protocol == "LEDENET":
+                # LEDENET devices support two white outputs for cold and warm. We set
+                # the second one here - if we're only setting a single white value,
+                # we set the second output to be the same as the first
+                if w2 is not None:
```

Comment:
Ok, I've restored that behavior. Though I still wonder if those devices could support the write mask (and be better off with it). Also, since 0x33 is in the rgbwprotocol list, am I assuming correctly that it's rgbwcapable? I added it to that list, let me know if that's wrong.